packages/client/components/ScopePhaseArea.tsx

```diff
@@ -61,43 +63,49 @@ const TabContents = styled('div')({
 const containerStyle = {height: '100%'}
 const innerStyle = {width: '100%', height: '100%'}
 
-const tabs = [
+const baseTabs = [
   {icon: <GitHubSVG />, label: 'GitHub'},
   {icon: <JiraSVG />, label: 'Jira'},
   {icon: <ParabolLogoSVG />, label: 'Parabol'}
 ]
 
 const ScopePhaseArea = (props: Props) => {
   const {meeting} = props
+  const {viewerMeetingMember} = meeting
+  if (!viewerMeetingMember) return null
```

Comment:
Why is `viewerMeetingMember` nullable?

Reply:
-1 React really doesn't like it when you return before all hooks 
https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

Reply:
Darn, I managed to miss those hooks below. Updated 👍